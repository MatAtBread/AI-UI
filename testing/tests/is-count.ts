/// <reference path="../test.env.d.ts"/>

for await (const x of Test.count()) {
    console.log("Count", x)
}

export { };