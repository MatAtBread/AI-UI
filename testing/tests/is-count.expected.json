[
  [1,"Count",0],
  [2,"Count",1],
  [3,"Count",2],
  [4,"Count",3],
  [5,"Count",4],
  [6,"Count",5],
  [7,"Count",6],
  [8,"Count",7],
  [9,"Count",8],
  [10,"Count",9]
]